projects/packages/forms/src/blocks/contact-form/variation-picker.js

```diff
@@ -77,26 +73,8 @@ export default function VariationPicker( { blockName, setAttributes, clientId, c
 			/>
 			<div className="form-placeholder__footer">
 				<Button variant="secondary" onClick={ () => setIsPatternsModalOpen( true ) }>
-					{ __( 'Explore Form Patterns', 'jetpack-forms' ) }
+					{ __( 'Browse form patterns', 'jetpack-forms' ) }
```

Comment:
This CTA lists exact same patterns as the icons but in more visual way + any reusable forms you've created.

So hence I wanted to get away from fuzzy "explore" term.